tools/all_questions.xml

```diff
@@ -4058,8 +4059,8 @@ Weitere relevante Schutzrechte können gewerbliche Schutzrechte wie Patente, Geb
 		<order>2</order>
 		<help lang="en"/>
 		<help lang="de"/>
-		<text lang="en">If **yes**, what are the reasons this dataset has to be preserved for the long-term?</text>
-		<text lang="de">Wenn **ja**, aus welchen Gründen müssen die Daten langfristig aufbewahrt werden?</text>
+		<text lang="en">For what reasons must the data set be preserved in the long-term?</text>
```

Comment:
"long-term" ändern in long term (s.o., Zeile 4034)

Reply:
ref: http://www.future-perfect.co.uk/grammar-tip/is-it-long-term-or-long-term/